composer.json

```diff
@@ -9,7 +9,8 @@
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "zendframework/zend-diactoros": "^1.6"
+        "zendframework/zend-diactoros": "^1.6",
+        "ext-json": "*"
```

Comment:
Does this break anything on non-Debian based systems?

(Iast I checked the JSON functions are a part of the PHP standard library, but removed from Debian and put into a separate package because the license for JSON is "non-free")

Reply:
@driusan I'm not sure. What distro would you recommend me test on to find out?

Reply:
Probably CentOS since it's what we use for prod servers.

Reply:
doesn't even have to be a Linux distro, a local Mac copy of php would be fine.. but CentOS is a good idea too.

Reply:
@driusan Well it doesn't seem to be an issue on macOS. I'll try testing on CentOS when I can get it setup.

Reply:
@driusan composer install does succeed on centos with the ext-json included in composer.json.